service/src/main/resources/db/changesets/20221014_workspace_activity_log_add_index.yaml

```diff
@@ -0,0 +1,12 @@
+databaseChangeLog:
+- changeSet:
+    id: workspace_activity_log_add_index
+    author: melchang
+    changes:
+    - createIndex:
```

Comment:
why do we not need this for the other table?

Reply:
Most of our other tables have primary keys, so they don't have a separate index. We do have a few where we have specific indexes.